doc/source/cluster/kubernetes/user-guides/upgrade-guide.md

```diff
@@ -28,7 +28,7 @@ Typically, while new fields are added to the KubeRay CRD in each release, KubeRa
 * KubeRay v1.0.0: CRD v1alpha1 and v1
 * KubeRay v1.1.0: CRD v1
 * KubeRay v1.2.2: CRD v1
-* KubeRay v1.3.0: CRD v1
```

Comment:
I assume you directly replaced every occurrence of the string 1.3.0 with 1.4.0. Would you mind reviewing all changes and then ping me for review later? Thanks.

Reply:
Fixed in https://github.com/ray-project/ray/pull/53884/commits/4bf4e39d14efaabdc832f2c96f6ae4b54aa5fb23

Reply:
Should we add compatibility for 1.4.X after this line?

https://github.com/ray-project/ray/blob/90933baf0cbff5b90b0e8f8f058ebe1ee382714e/doc/source/cluster/kubernetes/user-guides/upgrade-guide.md?plain=1#L19

Reply:
> Should we add compatibility for 1.4.X after this line?

yes

> And should we add v1.4.0 after this line?

yes